packages/store/tests/action-handler.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { ErrorHandler } from '@angular/core';
+import { TestBed, fakeAsync, tick } from '@angular/core/testing';
+
+import { Action } from '../src/decorators/action';
+import { State } from '../src/decorators/state';
+import { StateContext } from '../src/symbols';
+
+import { NgxsModule } from '../src/module';
+import { Store } from '../src/store';
+import { Actions } from '../src/actions-stream';
+import { NoopErrorHandler } from './helpers/utils';
+import { timer } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+describe('Action handlers', () => {
+  class TestAction {
+    static type = 'TestAction';
+    constructor(public payload?: any) {}
+  }
+
+  type _class = { new (...args: any[]) };
+
+  function setup(config: { stores: _class[] }) {
```

Comment:
you can choose ```Type<unknown>``` instead ```type _class = { new (...args: any[]) }```;

Reply:
Oh great! I didn't know about that type. Thanks

Reply:
Done